src/common/components/clusterDetail/OperatorsProgressItem.tsx

```diff
@@ -81,7 +81,11 @@ const OperatorsPopover: React.FC<OperatorsPopoverProps> = ({ operators, children
       bodyContent={
         <List className="operators-progress-item__operators-list">
           {operators.map((operator) => {
-            const status = operator.status || 'pending';
+            const status = operator.status
+              ? operator.status === 'available'
+                ? 'installed'
+                : operator.status
+              : 'pending';
```

Comment:
nit, in order to improve readability...
```suggestion
            const status = operator.status ?? 'pending'; // will fallback to 'pending' when operator.status is null or undefined
             if (operator.status === 'available') {
                operator.status = 'installed';
            }
```